extensions/amp-next-page/amp-next-page.md

```diff
@@ -163,82 +148,173 @@ Elements which are common across multiple pages can be programmatically
-  </tr>
-</table>
+_on the first document_
+
+```html
+<div class="sticky" next-page-replace="sticky-123">
+  <h2>The second sticky will replace me once you scroll past my page</h2>
+</div>
+```
 
-## Configuration spec
+_on the second document_
 
-The configuration defines the documents recommended by `amp-next-page` to
-the user as a JSON object.
+```html
+<div class="sticky" next-page-replace="sticky-123">
+  <h2>I replaced the first instance of my type (sticky-123)</h2>
+</div>
+```
 
-| Key                | Value                                                             |
-| ------------------ | ----------------------------------------------------------------- |
-| `pages` (required) | Ordered array of one or more page objects                         |
-| `hideSelectors`    | Optional array of string CSS selectors to hide in child documents |
+### Inline Configuration
+
+The inline configuration defines the documents recommended by `amp-next-page` to
+the user as an ordered array of page objects in JSON format.
 
 Each page object should have the following format:
 
-| Key                   | Value                                                                                                                                                            |
-| --------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `ampUrl` (required)   | String URL of the page. Must be on the same origin as the current document. URLs will automatically be rewritten to point to the Google AMP cache when required. |
-| `title` (required)    | String title of the page, will be used when rendering the recommendation box                                                                                     |
-| `imageUrl` (required) | String URL of the image to display in the recommendation box                                                                                                     |
+| Key                | Value                                                                                                                                                            |
+| ------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `url` (required)   | String URL of the page. Must be on the same origin as the current document. URLs will automatically be rewritten to point to the Google AMP cache when required. |
+| `title` (required) | String title of the page, will be used when rendering the recommendation box                                                                                     |
+| `image` (required) | String URL of the image to display in the recommendation box                                                                                                     |
+
```

Comment:
I've suggested a move for this section and migrated the information out of tables. (I don't really like how tables look on amp.dev, especially in mobile)